ui/components/src/UnitSelector/UnitSelector.tsx

```diff
@@ -105,17 +104,17 @@ export function UnitSelector({ value, onChange }: UnitSelectorProps) {
         }
       />
       <OptionsEditorControl
-        label="Decimal"
+        label="Decimals"
         control={
-          <Autocomplete
-            value={hasDecimalPlaces ? value.decimal_places ?? DEFAULT_DECIMAL_PLACES : 0}
-            options={DECIMAL_OPTIONS}
-            getOptionLabel={(option) => `${option}`}
-            renderInput={(params) => <TextField {...params} />}
-            onChange={handleDecimalChange}
-            disabled={!hasDecimalPlaces}
-            disableClearable
-          ></Autocomplete>
+          <TextField
+            type="number"
+            value={value.decimal_places ?? ''}
+            onChange={(e) => {
+              handleDecimalChange(e.target.value ? Number(e.target.value) : undefined);
+            }}
+            placeholder="Default"
```

Comment:
I like @foxbat07's idea for calling this "auto" instead of default, especially because it's slightly different behavior (significant digits instead of decimals).

Reply:
Yeah I like "Auto" too! (Also what Grafana uses). 

I'll save that for a different PR since it would require updating all the inputs (because they all say "Default" right now).